packages/template-chakra-storefront/src/components/_app-config/index.jsx

```diff
@@ -66,6 +76,22 @@ const AppConfig = ({children, locals = {}}) => {
     const passwordlessLoginCallbackURI = useMemo(() => passwordlessCallback, [passwordlessCallback])
     const defaultDnt = useMemo(() => locals.appConfig.dnt, [locals.appConfig.dnt])
 
+    /* eslint-disable react-hooks/rules-of-hooks */
+    const storeLocatorConfig =
+        SFDC_EXT_STORE_LOCATOR &&
```

Comment:
@kzheng-sfdc How does the template read this a variable from the pwa-kit-create-app? 

In the pwa-kit-create-app, we are already having an asser file _app-config.hbs to generate the block code based on the users selection of selectedPlugin, the code generated will not have this condition, it will follow the hbs template. 

I guess you want the monorepo template to be able to disable/enable this extension without going through app generation? I am not sure if this is the right way to get it right because with this we are having two versions of the same app for the same template for each extension. One for pwakit developers, and one for generated project, and of course the issue with React hook rules (which we should never turn off because it is a good practise when writing react).